frontend/src/deadlineList/components/userAvatar.jsx

```diff
@@ -13,10 +13,20 @@ const UserAvatar = ({ width, src, ...props }) => {
 			case 3:
 				translateX = 'transform -translate-x-3/2'
 		}
+		let border = 0
+		if (width <= 250) {
+			border = 0
```

Comment:
so with that change up instead of setting border to 0 just put border ..so it would have 1px width at least

Reply:
done